library/X509/Widget/Schedules.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/* Icinga Web 2 X.509 Module | (c) 2023 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\X509\Widget;
+
+use Icinga\Module\X509\Common\Links;
+use Icinga\Module\X509\Model\X509Schedule;
+use ipl\Html\BaseHtmlElement;
+use ipl\Html\Table;
+use ipl\I18n\Translation;
+use ipl\Orm\Query;
+use ipl\Web\Widget\Link;
+
+class Schedules extends BaseHtmlElement
+{
+    use Translation;
+
+    /** @var Query */
+    protected $schedules;
+
+    protected $tag = 'div';
+
+    protected $defaultAttributes = ['class' => 'job-schedules'];
+
+    public function __construct(Query $schedules)
+    {
+        $this->schedules = $schedules;
+    }
+
+    protected function assemble()
+    {
+        $table = new Table();
+        $table->getAttributes()->set('class', 'common-table table-row-selectable');
+
+        $modalOpener = ['data-icinga-modal' => true, 'data-no-icinga-ajax' => true];
+
+        /** @var X509Schedule $schedule */
+        foreach ($this->schedules as $schedule) {
+            $row = Table::tr(null, $modalOpener);
```

Comment:
Why do we need the modal data attributes on both tr and td?

Reply:
Not td, but `tr` and `a`. The `tr` tag is clickable when using a `table-row-selectable` class, and the name also contains the same link which redirects you to the same page as when you click the whole table row.